src/kaocha/repl.clj

```diff
@@ -81,11 +81,7 @@ These will particularly come in handy when developing plugins."}
    (config {}))
   ([extra-config]
    (let [config-file (:config-file extra-config "tests.edn")
-         config      (-> (config/load-config config-file)
-                         (config/merge-config (config/normalize extra-config))
-                         (cond-> #_config
-                           (.exists (io/file config-file))
-                           (assoc-in [:kaocha/cli-options :config-file] config-file)))
+         config      (config/load-config2 config-file)
```

Comment:
It seems this loses the `extra-config`? this is supposed to get merged into the config, so you can locally override bits of it.